modules/sor/sorV2/lib/poolsV3/weighted/weightedPool.ts

```diff
@@ -200,6 +240,7 @@ export class WeightedPoolV3 implements BasePoolV3 {
                     swapKind: SwapKind.GivenIn,
                 },
                 this.poolState,
+                this.hook,
```

Comment:
Same here about adding `hookState` to `swapGivenOut`.
Not sure if it applies, but ideally we should have a test that asserts this is being passed correctly.

Reply:
I have addressed this by naming the variable more clearly.